src/components/ContactGroups/helpers/groups.js

```diff
@@ -0,0 +1,12 @@
+export const checkIfGroupAlreadyExists = (
+  groupsAlreadyCreated,
+  groupToCreate
+) => {
+  let alreadyThisName = false
```

Comment:
How about something like this:

```
groupsAlreadyCreated.find(group => group.name.toLowerCase() === groupToCreate.name.toLowerCase()) !== undefined
```

I think it works the same and is a bit easier to read? Otherwise I would suggest we rename `alreadyThisName` to something like `isNameAlreadyUsed `